tools/codegen/gen_lazy_tensor.py

```diff
@@ -180,26 +181,6 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
         )),
     })
     # Generate headers for shape/dtype funcs for non-meta kernels
-    fm.write_with_template(f'{backend_key}ShapeInference.h', 'ShapeInference.h', lambda: {
```

Comment:
Was this duplicated further down - that's why we're deleting it?

Reply:
Yes, you are right.